package.json

```diff
@@ -43,7 +43,7 @@
     "codecov": "1.0.1",
     "commitizen": "2.9.5",
     "cross-env": "3.1.4",
-    "cz-lerna-changelog": "1.0.0",
+    "cz-lerna-changelog": "0.3.2",
```

Comment:
FYI: we cannot upgrade lerna yet, until `lerna-semantic-release` is published. They merged my PR https://github.com/atlassian/lerna-semantic-release/pull/42 but didn't release it yet 🙈